### PR TITLE
Avatar feature

### PR DESCRIPTION
**Pull request by mariyasada**
Hey, just done avatar component
https://61f3ec375736070008b4f88a--gallant-neumann-782cfd.netlify.app/htmlfiles/avatar

### CODE REVIEW

File: components.html [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+      </div>
+<!--------------------------------  side bar ends here-------------main content starts here--------------------------------- -->
+      <section class="main-content-container">
+         <div class="section1-container">
+             <h2 class="section-heading">
+                 <a href="">Alert</a>
+             </h2>
+
+             <p class="section-text-content">
+                A simple contextual message that notifies users about actions. Alerts are available for any length of text. Therea are lots of varieties of alert messages but alert-success,alert-primary, alert danger are frequently used.
```
Comment (harshitadaswani) [inline components.html]: You can fix the typo of **there**

File: components.html [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+             </p>
+
+             <hr>
+             <h3 class="section-sub-heading">Types of alert</h3>
+             <hr>
+
+             <p class="section-text-content">There are availabe in five types: primary alert, secondary alert,success alert, warning alert, red alert and dark alert. To use this alert just add the class name to alert box.</p>
```
Comment (harshitadaswani) [inline components.html]: again typo of there to **they**

Comment (mariyasada) [reply components.html]: done the correction

File: stylesheets/avatar/avatar.css [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+}
+.code-snippet img
+{   
+    /* display:none; */
+    max-height:80vh;
+    max-width:60vw;
+    object-fit: cover;
+}
+/* media query */
+@media only screen and (min-width:500px )and(max-width:1045px) {
```
Comment (sks-21) [inline stylesheets/avatar/avatar.css:129]: The code looks nice. Just one suggestion, it is a good practice to add media queries at the end.

Comment (mariyasada) [reply stylesheets/avatar/avatar.css:129]: At the end of project?

**Pull request merged**